google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/MerchantCenterDynamicRemarketing.java

```diff
@@ -0,0 +1,384 @@
+      System.out.printf("Created ad group with resource name '%s'.%n", adGroupResourceName);
+      return adGroupResourceName;
+    }
+  }
+
+  /**
+   * Creates the responsive display ad.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID in which to create criterion.
+   * @param adGroupResourceName the campaign resource name.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void createAd(
+    GoogleAdsClient googleAdsClient,
+    long customerId,
+    String adGroupResourceName) throws IOException {
+    String marketingImageUrl = "https://goo.gl/3b9Wfh";
+    String marketingImageName = "Marketing Image";
+    String marketingImageResourceName =
+      uploadAsset(googleAdsClient, customerId, marketingImageUrl, marketingImageName);
+    String logoImageName = "Logo Image";
+    String logoImageUrl = "https://goo.gl/mtt54n";
+    String logoImageResourceName =
+      uploadAsset(googleAdsClient, customerId, logoImageUrl, logoImageName);
+
+    // Creates the responsive display ad info object.
+    ResponsiveDisplayAdInfo responsiveDisplayAdInfo = ResponsiveDisplayAdInfo.newBuilder()
+      .addMarketingImages(AdImageAsset.newBuilder()
+        .setAsset(StringValue.of(marketingImageResourceName))
+        .build())
+      .addSquareMarketingImages(AdImageAsset.newBuilder()
+        .setAsset(StringValue.of(logoImageResourceName))
+        .build())
+      .addHeadlines(AdTextAsset.newBuilder().setText(StringValue.of("Travel")).build())
+      .setLongHeadline(
+        AdTextAsset.newBuilder().setText(StringValue.of("Travel the World")).build())
+      .addDescriptions(
+        AdTextAsset.newBuilder().setText(StringValue.of("Take to the air!")).build())
+      .setBusinessName(StringValue.of("Interplanetary Cruises"))
+      // Optional: Call to action text.
+      // Valid texts: https://support.google.com/adwords/answer/7005917
+      .setCallToActionText(StringValue.of("Apply Now"))
+      // Optional: Creates a logo image and set it to the ad.
+      .addLogoImages(
+        AdImageAsset.newBuilder().setAsset(StringValue.of(logoImageResourceName)).build())
+      // Optional: Creates a logo image and set it to the ad.
+      .addSquareLogoImages(
+        AdImageAsset.newBuilder().setAsset(StringValue.of(logoImageResourceName)).build())
+      // Whitelisted accounts only: Sets color settings using hexadecimal values.
+      // Sets allowFlexibleColor to false if you want your ads to render by always
+      // using your colors strictly.
+      /*
+      .setMainColor(StringValue.of("#0000ff"))
+      .setAccentColor(StringValue.of("#ffff00"))
+      .setAllowFlexibleColor(BoolValue.of(false))
+      */
+      // Whitelisted accounts only: Sets the format setting that the ad will be
+      // served in.
+      /*
+      .setFormatSetting(DisplayAdFormatSetting.NON_NATIVE)
```

Comment:
i think the convention is using // to comment out the optional codes, instead of /* */
as L63 in "AddHotelAdGroupBidModifiers", L66 in "AddAdGroupBidModifier" samples

Reply:
This is how we did in the the previous version of this example. I assume it is because the actual comments have the `//` while the code that is commented out is surrounded by `/* */` which makes sense to me.